routes/web.php

```diff
@@ -44,11 +44,15 @@
     Route::get('/logout', 'LoginController@logout')->name('logout');
     Route::get('/dashboard', 'DashboardController@dashboard')->name('dashboard')->middleware('checkLoginAdmin');
     //Product
-    Route::get('/product', 'ProductController@listAllProducts')->name('listAllProducts');
+    Route::get('/product', 'ProductController@index')->name('index');
+    Route::get('/create', 'ProductController@create')->name('create');
     Route::post('/store', 'ProductController@store')->name('store');
     Route::get('/edit/{product}', 'ProductController@editProduct')->name('editProduct');
     Route::put('/update/{id}', 'ProductController@update')->name('update');
     Route::get('/exportProduct', 'ProductController@exportProduct')->name('exportProduct');
     Route::post('/importProduct', 'ProductController@importProduct')->name('importProduct');
-    Route::delete('/destroyProduct/{id}', 'ProductController@destroyProduct')->name('destroyProduct');
+    Route::delete('/delete/{id}', 'ProductController@destroy')->name('destroy');
```

Comment:
Should name clearly
EX:
    '/update/{id}' => '/product/{id}/update'
    '/delete/{id}' => '/product/{id}/delete'
But I suggest using resource:
  Route::resource('products', 'ProductController')->middleware('checkLoginAdmin');